x/accounts/proto/cosmos/accounts/defaults/base/v1/base.proto

```diff
@@ -4,10 +4,12 @@ package cosmos.accounts.defaults.base.v1;
 
 option go_package = "cosmossdk.io/x/accounts/defaults/base/v1";
 
+import "google/protobuf/any.proto";
+
 // MsgInit is used to initialize a base account.
 message MsgInit {
-  // pub_key defines the secp256k1 pubkey for the account.
-  bytes pub_key = 1;
+  // pub_key defines a pubkey for the account arbitrary encapsulated.
+  google.protobuf.Any pub_key = 1;
```

Comment:
why do we need an Any here? What would be the improvement over just bytes?

Reply:
Probably so that you know from the type url what is the pub key type